### PR TITLE
Remove "--docker-metrics" functionality.

### PR DESCRIPTION
**Pull request by yabinmeng**

### CODE REVIEW

File: engine-cli/src/main/resources/docker-metrics.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # docker-metrics
 
+```
+NOTE: This feature has been removed and is no longer available.
```
Comment (jshook) [inline engine-cli/src/main/resources/docker-metrics.md:4]: We should change this to "... and will be replaced by a simpler mechanism soon." or something similar. Some of our biggest users rely on this and will be expecting a replacement, which we plan to provide.

**Review by jshook — CHANGES_REQUESTED** [review]
This is a good first step. Only one requested change before merging.

File: engine-cli/src/main/resources/docker-metrics.md (hunk 1 above)
Comment (yabinmeng) [reply engine-cli/src/main/resources/docker-metrics.md]: Changed

**Review by jshook — APPROVED** [review]
excellent, merging

**Pull request merged**